toolbox/providers/gcp/gcp-advanced.md

```diff
@@ -1,40 +1,46 @@
-# # Nauta on Google Cloud Platform - Advanced options
+# # Nauta on Google Cloud* Platform: Advanced Options
 
 This document contains information referring to [Nauta on Google Cloud Platform - Getting Started](gcp.md).
 
-## Overall cluster architecture
+## Overall Cluster Architecture
+
 GCP Nauta Kubernetes instance consists of several parts:
-- gateway - bastion host to access to cluster
-- test node (optional) - bastion host with test software to do continuous tests
-- nfs server - internal server with nfs based resources
-- gke cluster - kubernetes cluster.
 
-Please refer to terraform definition files (in directory terraform/templates) to see detailed configuration (networking,
```

Comment:
Missed this one. Should be....

Refer to terraform definition files (in directory terraform/templates) to see detailed configuration (networking, resources, etc.)